readthedocs/oauth/services/github.py

```diff
@@ -100,37 +99,30 @@ def create_repository(self, fields, privacy=None, organization=None):
         ]):
 
             repo, _ = RemoteRepository.objects.get_or_create(
-                remote_id=fields['id'],
-                vcs_provider=self.vcs_provider_slug
-            )
-            remote_repository_relation = repo.get_remote_repository_relation(
-                self.user, self.account
+                remote_id=str(fields["id"]),
+                vcs_provider=self.vcs_provider_slug,
             )
 
-            # It's possible that a user has access to a repository from within
-            # an organization without being member of that organization
-            # (external contributor). In this case, the repository will be
-            # listed under the ``/repos`` endpoint but not under ``/orgs``
-            # endpoint. Then, when calling this method (``create_repository``)
-            # we will have ``organization=None`` but we don't have to skip the
-            # creation of the ``RemoteRepositoryRelation``.
-            if repo.organization and organization and repo.organization != organization:
-                log.debug(
-                    'Not importing repository because mismatched orgs.',
-                    repository=fields['name'],
+            owner_type = fields["owner"]["type"]
+            organization = None
+            if owner_type == "Organization":
+                # NOTE: We shouldn't create a remote relationship with the current user here,
+                # since the user can have access to the repository, but not to the organization.
```

Comment:
```suggestion
                # We are not creating a remote relationship between the current user and the organization,
                # since the user can have access to the repository, but not to the organization.
```